website/patches/docusaurus-plugin-internaldocs-fb+1.14.0.patch

```diff
@@ -0,0 +1,80 @@
+diff --git a/node_modules/docusaurus-plugin-internaldocs-fb/docusaurus-preset.js b/node_modules/docusaurus-plugin-internaldocs-fb/docusaurus-preset.js
+index d85567a..524e56e 100644
+--- a/node_modules/docusaurus-plugin-internaldocs-fb/docusaurus-preset.js
++++ b/node_modules/docusaurus-plugin-internaldocs-fb/docusaurus-preset.js
+@@ -91,6 +91,14 @@ function preset(context, opts = { docs: {} }) {
+     result.themes.push([require.resolve('.'), opts]);
+     if (!((_d = (_c = context.siteConfig) === null || _c === void 0 ? void 0 : _c.themeConfig) === null || _d === void 0 ? void 0 : _d.algolia)) {
+         result.plugins.push(require.resolve('docusaurus-lunr-search'));
++        result.themes.push(function() {
++          return {
++            name: 'docusaurus-fb-search-theme',
++            getThemePath: () => {
++              return path_1.default.resolve(__dirname, './theme-search');
++            },
++          };
++        });
```

Comment:
@antonk52 need to create a separate conditional theme for lunr SearchBar

Reply:
This is really impressive. I apply these changes to the original theme and create a new version this week. Sorry about having to check out the output code.